test/test_artifacts/v1/langchain-aws.test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG SAGEMAKER_DISTRIBUTION_IMAGE
+FROM $SAGEMAKER_DISTRIBUTION_IMAGE
+
+ARG MAMBA_DOCKERFILE_ACTIVATE=1
+
+CMD ["python", "-c", "import langchain_aws"]
```

Comment:
Can we add more tests

Reply:
Thank you for giving this PR a look @balajisankar15. Is this blocking / could we add more tests in the follow-up PR?

Reply:
@balajisankar15 
We can add more tests around importing other sub-modules to test their importability. Anything other than that requires an elaborate setup with boto3 credentials and service endpoint infrastructure (bedrock, kendra, vectorstores etc.) to test each of these components, which we don't plan to do for this library. 
We will update the PR with these and open for a review. Thanks.